package.json

```diff
@@ -28,19 +23,34 @@
   },
   "homepage": "https://github.com/jcoreio/mutate#readme",
   "devDependencies": {
-    "@jedwards1211/js-toolchain": "^2.0.6",
-    "husky": "^4.3.0"
+    "@typescript-eslint/eslint-plugin": "^5.60.0",
+    "@typescript-eslint/parser": "^5.60.0",
+    "chai": "^4.3.7",
+    "eslint": "^8.43.0",
+    "mocha": "^10.2.0",
+    "typescript": "^5.1.0"
   },
   "dependencies": {
+    "@babel/runtime": "^7.18.6",
+    "@jcoreio/toolchain": "1.0.0-beta.10",
```

Comment:
@forrest-keller I just published v1.0.0, if you want to update these

Reply:
oh and actually I think there are errors when importing `@jcoreio/toolchain-semantic-release/release.config.cjs` in the beta version